packages/edit-navigation/src/hooks/use-navigation-editor.js

```diff
@@ -73,7 +75,7 @@ export default function useNavigationEditor() {
 			force: true,
 		} );
 		if ( didDeleteMenu ) {
-			setSelectedMenuId( null );
+			setSelectedMenuId( 0 );
```

Comment:
Just checking...is it safe to set this to `0`? What if there are no menus left?

Reply:
If there are no menus left, the screen will display create menu placeholder. I'm using `0` here so we can have a single type for selectedMenuId. But having `null` as in no menu was selected also makes sense to me.